packages/@react-spectrum/form/src/Form.tsx

```diff
@@ -18,10 +18,14 @@ import React, {useContext} from 'react';
 import {SpectrumFormProps} from '@react-types/form';
 import styles from '@adobe/spectrum-css-temp/components/fieldlabel/vars.css';
 
-let FormContext = React.createContext<SpectrumLabelableProps>({});
+let FormContext = React.createContext<SpectrumLabelableProps>(null);
 export function useFormProps<T extends SpectrumLabelableProps>(props: T): T {
   let ctx = useContext(FormContext);
-  return {...ctx, ...props};
+  if (ctx) {
+    return {...ctx, ...props};
```

Comment:
you should be able to spread null and it just doesn't do anything `{...null} -> {}`
where'd this change come from? just curious

Reply:
Yeah, this was a hack to know if we are inside a form or not. If `useFormProps` returns the original object, then we know there was no context. Also, this should be faster in non-form cases because no extra objects are created.